scripts/prepare_release.sh

```diff
@@ -43,7 +43,7 @@ fi
 git checkout -b release_v${VERSION}
 
 # Changes the version for release and creates the commits/tags.
-echo | ./gradlew release
+./gradlew release -Prelease.releaseVersion=${VERSION}
```

Comment:
`echo |` is unnecessary only when passing `-Prelease.useAutomaticVersion=true`. (It's for not having to press ENTER on the command line.)